module/zfs/spa_misc.c

```diff
@@ -426,6 +426,8 @@ spa_load_note(spa_t *spa, const char *fmt, ...)
 
 	zfs_dbgmsg("spa_load(%s, config %s): %s", spa->spa_name,
 	    spa->spa_trust_config ? "trusted" : "untrusted", buf);
+
+	spa_import_progress_set_notes(spa_guid(spa), buf);
```

Comment:
Won't this get into dbgmsg twice?

Reply:
Ah, yes good catch.  Will change to use spa_import_progress_set_notes_nolog()